code/core/src/preview-api/modules/store/StoryStore.ts

```diff
@@ -78,9 +77,7 @@ export class StoryStore<TRenderer extends Renderer> {
   ) {
     this.storyIndex = new StoryIndexStore(storyIndex);
 
-    this.projectAnnotations = normalizeProjectAnnotations(
-      composeConfigs([...getCoreAnnotations(), projectAnnotations])
-    );
```

Comment:
Cab you explain this?

Reply:
`composeConfigs` internally already composes the coreAnnotations. They were applied twice!